wcivf/apps/parties/templates/parties/party_list.html

```diff
@@ -12,7 +12,10 @@
 <section class="columns large-12">
   <div class="card">
     <div class="row">
-      <h2>UK Political parties</h2>
+      <h2>UK political parties</h2>
+      The following is a list of all political parties represented by candidates in Democracy Club's UK election database (est. 2015). Party names and emblems are taken from the [Electoral Commission’s](http://www.electoralcommission.org.uk/) register of political parties.
```

Comment:
For both of these lines, could you wrap them in a `<p>` tag (that is `<p>[words]</p>` for each line?

This isn't needed if the text is in a markdown block, but is needed if it's just HTML (e.g, not in a `{% markdown %}` block)

Reply:
(Let me know if the GitHub UI makes this hard and I can do it)

Reply:
Thanks, I think I've done this? I assumed this needed to be a commit, not a PR...